geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/data/AbstractMemoryOverheadIntegrationTest.java

```diff
@@ -239,7 +276,7 @@ private void measureAndCheckPerEntryOverhead(AddEntryFunction addEntry, Measurem
     // Put some entries to make sure we initialize any constant size data structures. We are
     // just trying to measure the cost of each add entry operation.
     for (int i = 0; i < WARM_UP_ENTRY_COUNT; i++) {
-      String uniqueString = String.format("warmup-%10d", i);
+      String uniqueString = String.format("warmup-%010d", i);
```

Comment:
It was deliberate, because it could produce string with spaces in them which confused the earlier version of the LPUSh executor. It's no longer necessary thanks to later development. Good catch!